OpenRA.Mods.Common/Scripting/Global/EffectGlobal.cs

```diff
@@ -32,5 +35,17 @@ public void Flash(string type = null, int ticks = -1)
 				if (fpe.Info.Type == type)
 					fpe.Enable(ticks);
 		}
+
+		[Desc("Dynamically adjusts the `GlobalLightingPaletteEffect` trait.")]
+		public void ChangeLighting(double red, double green, double blue, double ambient)
```

Comment:
@GraionDilach: Limitations of the .NET api mean we can't expose the standard Color type directly.  That's why I suggested we produce an api for creating a color object from RGB values.